.github/scripts/mkdocs-hooks.py

```diff
@@ -0,0 +1,21 @@
+import os
```

Comment:
this is how the supporting assets (i.e. - images) for a pattern are copied over into the built doc site. If you see the current Kubecost pattern, it works in markdown on GitHub, but not on the mkdoc site - I'll fix that after this PR using this code below